geozero-shp/src/reader.rs

```diff
@@ -3,9 +3,10 @@ use crate::shx_reader::{read_index_file, ShapeIndex};
 use crate::{header, Error};
 use geozero::{FeatureProcessor, FeatureProperties, GeomProcessor};
 use std::fs::File;
-use std::io::{BufReader, Read};
+use std::io::{BufReader, Read, Seek};
 use std::iter::FusedIterator;
 use std::path::Path;
+pub use dbase::FieldInfo;
```

Comment:
`pub` is not strictly necessary here. Did you deliberately make it pub?

Reply:
It was on purpose as I thought it would be good to reexport the FieldInfo and FieldType so that users would not need to declare the dbf crate just for those 2 types. 
And I wanted to match on FieldType in the example however I discovered that FIeldType is in a private module. Hence the issue I referred to.